libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.kt

```diff
@@ -246,7 +250,7 @@ open class PushService {
             getRestClient(account)?.let { restClient ->
                 val apiVersion = ApiVersionStrings.getVersionNumber(SalesforceSDKManager.getInstance().appContext)
                 // TODO remove once MSDK default api version is 61 or greater
-                if (apiVersion.compareTo("v61.0") >= 0) {
+                if (apiVersion >= "v61.0") {
```

Comment:
This was the last inspector warning in this source, so I went in for a little clean up.